readme.md

```diff
@@ -1,5 +1,4 @@
 # sbt-scalafmt
-[![Build Status](https://travis-ci.org/scalameta/sbt-scalafmt.svg?branch=master)](https://travis-ci.org/scalameta/sbt-scalafmt)
```

Comment:
Instead of changing this I just removed it as it's not longer valid since we don't build on Travis and it's instead just as easy to see the ✅ or ❌ next to the commit when you're looking at the repo.